bench/coinSelection.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const assert = require('bsert');
+const random = require('bcrypto/lib/random');
+const bench = require('./bench');
+const WalletDB = require('../lib/wallet/walletdb');
+const MTX = require('../lib/primitives/mtx');
+const Input = require('../lib/primitives/input');
+const Output = require('../lib/primitives/output');
+const Script = require('../lib/script/script');
+const {rimraf, testdir} = require('../test/util/common');
+
+const ITERATIONS = 1;
+
+(async () => {
+  for (const memory of [true, false]) {
+    console.log(`Memory: ${memory}`);
+
+    const location = testdir('coinselection');
+
+    // Create walletDB and wallet
+    const wdb = new WalletDB({
+      memory,
+      location
+    });
+    await wdb.open();
+    const wallet = await wdb.create();
```

Comment:
it would be better to pass null as param rather than passing nothing as null is better than undefined.

Reply:
It won't make any significant improvements or difference.